playwright/src/test/java/com/microsoft/playwright/ServerWithClientCertificate.java

```diff
@@ -149,7 +149,7 @@ public void handle(HttpExchange exchange) throws IOException {
           cert.getSubjectX500Principal().getName(), cert.getIssuerX500Principal().getName()));
         exchange.sendResponseHeaders(403, 0);
       }
-    } else {
+    } catch (SSLPeerUnverifiedException e) {
```

Comment:
What about NPE and index out of range exceptions which may happen if there  are no certs?

Reply:
happy to add it back but for some reason it wasn't used, so I thought removing it is better then.